packages/salesforcedx-vscode-core/src/diagnostics/diagnostics.ts

```diff
@@ -13,6 +13,18 @@ import {
 import * as path from 'path';
 import * as vscode from 'vscode';
 
+export function getFileUri(
+  workspacePath: string,
+  filePath: string,
+  defaultErrorPath: string
+): string {
+  const resolvedFilePath = filePath.includes(workspacePath)
+    ? filePath
+    : path.join(workspacePath, filePath);
+  // source:deploy sometimes returns N/A as filePath
+  return filePath === 'N/A' ? defaultErrorPath : resolvedFilePath;
```

Comment:
Should we track when this happens?

Reply:
nit: make 'N/A' a constant

Reply:
Not sure if we should track this.  I posed a question internally in Slack to see if this "N/A" response is still relevant...